Source/Core/VideoBackends/D3D/main.cpp

```diff
@@ -37,6 +37,31 @@ std::string VideoBackend::GetDisplayName() const
   return _trans("Direct3D 11");
 }
 
+std::optional<std::string> VideoBackend::GetWarningMessage() const
+{
+  std::optional<std::string> result;
+
+  // If user is on Win7, show a warning about partial DX11.1 support
+  // This is being called BEFORE FillBackendInfo is called for this backend,
+  // so query for logic op support manually
+  bool supportsLogicOp = false;
+  if (D3DCommon::LoadLibraries())
+  {
+    supportsLogicOp = D3D::SupportsLogicOp(g_Config.iAdapter);
+    D3DCommon::UnloadLibraries();
+  }
+
+  if (!supportsLogicOp)
+  {
+    result = _trans("Direct3D 11 renderer requires support for features not supported by your "
+                    "system configuration. This most likely means that you are using Windows 7. "
```

Comment:
What do you think about changing "This most likely means that you are using Windows 7" to "This is most likely because you are using Windows 7"? So that it will be more clear that Windows 7 is the problem.

(Also, this problem should apply to Windows Vista as much as it applies to Windows 7, but I'm not sure if that's worth mentioning considering how few people use Vista compared to 7...)

Reply:
IF there are cards/drivers on Win8+ which don't support logic op then the same error will show. Do you think because of this possibility we should state something as strongly as "you ARE using Windows 7"?

Reply:
No, I think we should keep the "most likely". My suggestion is just to change the "means" to "is because".

Reply:
I agree. Will change.